website/docs/howto.md

```diff
@@ -137,17 +137,3 @@ You can list all constraints in a cluster with the following command:
 kubectl get constraints
 ```
 
-### Input Review
```

Comment:
We have existing issues that link or reference this section already. Perhaps leave the section and point to the new page?